import-export-cli/box/resources/init/document.yaml

```diff
@@ -0,0 +1,10 @@
+type: document
+version: v4
```

Comment:
Shall we change this to v4.0.0 like in other places?

Reply:
ack aiya